src/inverseRobot.js

```diff
@@ -8,6 +8,24 @@
 
 function inverseRobot(robot) {
   // write code here
+
+  const inverseRobotResult = {};
+  const arrayOfKeys = [];
+
+  for (const key in robot) {
+    inverseRobotResult[robot[key]] = key;
+    arrayOfKeys.push(robot[key]);
+  }
+
+  arrayOfKeys.sort();
+
+  for (let i = 0; i < arrayOfKeys.length; i++) {
+    if (arrayOfKeys[i] === arrayOfKeys[i + 1]) {
+      return null;
+    }
+  }
```

Comment:
Try to simplify your solution and use only 1 loop.
P.S. To check if key already present in inversedRobot, you can use `hasOwnProperty`